railway_ipc.gemspec

```diff
@@ -38,4 +38,10 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "google-protobuf", "~> 3.9"
   spec.add_dependency "sneakers", "~> 2.3.5"
   spec.add_dependency "bunny", "~> 2.2.0"
+
+  # Setup for testing Rails type code within mock Rails app
+  spec.add_development_dependency "rails", "~> 5.0.7"
```

Comment:
Ahhh, this is instead of updating the gemfile, isn't it.

Reply:
Yea, so the Gemfile Is within the Rails app, but this lives within the actual gem. I only added the Gems here that were necessary for the rails app to live in the /spec/support folder.